packages/connect-web/src/popup/index.ts

```diff
@@ -447,4 +460,17 @@ export class PopupManager extends EventEmitter {
             this.channel.postMessage(message);
         }
     }
+
+    public emitClosed() {
+        if (this.settings?.useCoreInPopup) {
+            // When popup is closed we should create a not-real response as if the request was interrupted.
+            // Because when popup closes and TrezorConnect is living there it cannot respond, but we know
+            // it was interrupted so we safely fake it.
```

Comment:
good point, but in cases beforeunload event is fired from popup:
```
const handleWindowBeforeUnload = (_e: BeforeUnloadEvent) => {
    if (getState().core) {
        const core = ensureCore();
        core.handleMessage({
            type: POPUP.CLOSED,
            payload: null,
        });
    }
};
```
 don't we get this emitted twice? probably nope, its the line 311 in this file right?